README.md

```diff
@@ -1,17 +1,20 @@
-# Go Bindings for GLFW 3.1
-
-* **ATTENTION:** As of GLFW 3.1 we break API. See Changelog below.
-* See [here](http://godoc.org/github.com/go-gl/glfw/v3.1/glfw) for documentation.
-* You can help by submitting examples to [go-gl/examples](http://github.com/go-gl/examples).
+# GLFW 3.1 for Go [![GoDoc](https://godoc.org/github.com/go-gl/glfw/v3.1/glfw?status.svg)](https://godoc.org/github.com/go-gl/glfw/v3.1/glfw)
 
 ## Installation
 
-* Installation is easy, just `go get github.com/go-gl/glfw/v3.1/glfw` and be done (*GLFW sources are included so you don't have to build GLFW on your own*)!
-* Go 1.4 is required on Windows (otherwise you must use MinGW v4.8.1 exactly, see [Go issue 8811](https://code.google.com/p/go/issues/detail?id=8811)).
+* GLFW C library source is included and built automatically as part of the Go package. But you need to make sure you have dependencies of GLFW:
+	* On OS X, you need Xcode for required headers and libraries.
+	* On Linux, you need `libgl1-mesa-dev` and `xorg-dev` packages.
```

Comment:
xorg-dev is a meta package for debian/ubuntu derivatives.
Instead of "on linux" I would prefer "on debian/ubuntu derivatives"
Here is the list of packages that xorg-dev installs: https://packages.debian.org/sid/x11/xorg-dev